graylog2-server/src/test/java/org/graylog/testing/ldap/OpenLDAPContainer.java

```diff
@@ -55,7 +55,8 @@ public static OpenLDAPContainer createWithTLS() {
                 .withFileSystemBind(LDAPTestUtils.testTLSCertsPath("server-cert.pem"), CONTAINER_CERTS_PATH + "/server-cert.pem", BindMode.READ_ONLY)
                 .withFileSystemBind(LDAPTestUtils.testTLSCertsPath("server-key.pem"), CONTAINER_CERTS_PATH + "/server-key.pem", BindMode.READ_ONLY)
                 .withFileSystemBind(LDAPTestUtils.testTLSCertsPath("CA-cert.pem"), CONTAINER_CERTS_PATH + "/CA-cert.pem", BindMode.READ_ONLY)
-                .withFileSystemBind(LDAPTestUtils.testTLSCertsPath("dhparam.pem"), CONTAINER_CERTS_PATH + "/dhparam.pem", BindMode.READ_ONLY);
+                .withFileSystemBind(LDAPTestUtils.testTLSCertsPath("dhparam.pem"), CONTAINER_CERTS_PATH + "/dhparam.pem", BindMode.READ_ONLY)
+                .withExposedPorts(TLS_PORT);
```

Comment:
@juju2112 It will work once you add the `PORT` as well. :slightly_smiling_face: The image exposes two ports and all of them will be requested for the TLS tests.

```suggestion
                .withExposedPorts(PORT, TLS_PORT);
```

Reply:
Thanks, that worked! It's always something dumb like that. :D